andela/ah-frontend-lannister
# #160617650 List Bookmarked articles

#### What does this PR do?
- Enables users to view a list of articles they have bookmarked
#### Description of Task to be completed?
- Enable authenticated users to view bookmarked articles
#### How should this be manually tested?
- When you click on profile picture on the navbar a drop-down menu appears. 
- Select bookmarks
#### What are the relevant pivotal tracker stories?
[#160617650]

#### Screenshoot
<img width="1408" alt="screen shot 2018-11-22 at 16 50 12" src="https://user-images.githubusercontent.com/1590202/48906830-c268cd80-ee76-11e8-9908-33bc1c4d8add.png">
